presets/minimal/eip-7732.yaml

```diff
@@ -0,0 +1,9 @@
+# Minimal preset - EIP7732
+
+# Execution
+# ---------------------------------------------------------------
+# 2**1(= 2)
+PTC_SIZE: 2
```

Comment:
Here we have 2 out of 64 validators. But because validators are split up into SLOTS_PER_EPOCH committees, only 8 of them vote on each slot. So, 25% is PTC which does not count in the numerator but counts in the denumerator. If there is some randomness involves, may slow down finality (but not fully break it at least).